File: Projects/TransportathonHackathon.API/TransportathonHackathon.Domain/Entities/Driver.cs
using Core.Persistence.Entities;

namespace TransportathonHackathon.Domain.Entities
{
    public class Driver : Entity
    {
        public Guid EmployeeId { get; set; }

        public virtual Employee Employee { get; set; }
        public virtual DriverLicense? DriverLicense { get; set; }
        public virtual ICollection<Vehicle>? Vehicles { get; set; }
    }
}

File: Projects/TransportathonHackathon.Clients/angular-ui/src/app/models/entities/driver.ts
import { BaseEntity } from 'src/app/core/models/entities/entity';
import { DriverLicense } from './driverLicense';
import { Employee } from './employee';

export interface Driver extends BaseEntity {
  employeeId: string;
  driverLicense: DriverLicense | null;
  employee: Employee;
}
